### DISCUSSION TITLE
how to update team information?

### DISCUSSION THREAD

**Discussion by voladorlu** [snap-stanford/ogb#334] [PCQM4M-LSC]
Hi, May I ask how to update the team member?

**Comment by weihua916** [comment]
We cannot update team members, as we clearly mentioned in our [Google form](https://docs.google.com/forms/d/e/1FAIpQLSe0-xCtMm5fe87WEwFOn9M75Y5vI1Cq73PZ9TURyF4Og1hxuw/viewform): "Important: You *cannot* add/delete any members once you registered your team. Please register your team after all the members are finalized."

**Reply by voladorlu** [reply]
Hi @weihua916 Thanks so much for your answer. Well, I just mistakenly registered a team without complete list of members.

**Reply by weihua916** [reply]
Ok, we can make a special exception. Please email me your team information (team name and password), and I can delete it.
